public/angular/templates/settings.html

```diff
@@ -1,10 +1,39 @@
 <div class="bs-component">
-  <div class="panel panel-info">
-    <div class="panel-heading">
-      <h3 class="panel-title">Settings</h3>
-    </div>
-    <div class="list-group">
-      <a href ui-sref="users" class="list-group-item">Users</a>
-    </div>
-  </div>
+  <ul class="breadcrumb">
+    <li class="active">Settings</li>
+    <li><a ui-sref="users">Users</a></li>
+  </ul>
+</div>
+
+<div ng-controller="FetchingController">
+  <table class="table">
+    <tbody>
+      <tr>
+        <td>
+          <h5>Fetching</h5>
```

Comment:
The table is presently too wide I think. Can you make it substantially narrower. Also usually the left column is narrower than the right in tables like this. Can we see how that looks?